pkg/backup/backup/backup_manager.go

```diff
@@ -250,19 +250,30 @@ func (bm *backupManager) makeExportJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 
 	return job, "", nil
 }
+
 func (bm *backupManager) makeBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, string, error) {
 	ns := backup.GetNamespace()
 	name := backup.GetName()
 
-	envVars, reason, err := backuputil.GenerateStorageCertEnv(ns, backup.Spec.StorageProvider, bm.secretLister)
+	envVars, reason, err := backuputil.GenerateTidbPasswordEnv(ns, name, backup.Spec.From.SecretName, bm.secretLister)
+	if err != nil {
+		return nil, reason, err
+	}
+
+	storageEnv, reason, err := backuputil.GenerateStorageCertEnv(ns, backup.Spec.StorageProvider, bm.secretLister)
 	if err != nil {
 		return nil, reason, fmt.Errorf("backup %s/%s, %v", ns, name, err)
 	}
 
+	envVars = append(envVars, storageEnv...)
+
 	args := []string{
 		"backup",
 		fmt.Sprintf("--namespace=%s", ns),
 		fmt.Sprintf("--backupName=%s", name),
+		fmt.Sprintf("--host=%s", backup.Spec.From.Host),
+		fmt.Sprintf("--port=%d", backup.Spec.From.Port),
+		fmt.Sprintf("--user=%s", backup.Spec.From.User),
```

Comment:
Actually, these info can be got from the backup CR, no need to pass it as command-line options.

Reply:
You may also have to update `pkg/backup/util/util.go:func ValidateBackup` since BR requires the `from.host` now.